tests/conftest.py

```diff
@@ -4,7 +4,7 @@
 
 @pytest.fixture(scope="module")
 def test_client():
-    app = create_app(test_config=True)
+    app = create_app(True)
```

Comment:
app = create_app(test_config=True)

For Line 7 can you change it like this? I think you forgot to change it back oops